_projects/SP19-Elemental-Experience.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: Elemental Experience
+desc: |
+ a zenful experience that allows you to control the elements of Earth, Water, Fire, and Air.
+github: https://github.com/Greenninja4/VR_Group_16
```

Comment:
GitHub link doesn't work, is your repository private?

Reply:
The repo was private, but it should be public now